src/roadstop.cpp

```diff
@@ -353,14 +368,14 @@ void RoadStop::Entry::Rebuild(const RoadStop *rs, int side)
 {
 	assert(HasBit(rs->status, RSSFB_BASE_ENTRY));
 
-	DiagDirection dir = GetRoadStopDir(rs->xy);
+	Axis axis = GetDriveThroughStopAxis(rs->xy);
 	if (side == -1) side = (rs->east == this);
```

Comment:
Not your code, but this seems to be converting an `int` to a `bool` that you pass to `GetEntryDirection()`. Maybe out of scope but this is ugly (and I don't even know how it's allowed). 🙂

Reply:
I know it's ugly, but... conflicts: https://github.com/OpenTTD/OpenTTD/commit/b49f13ba457329fecb1c05291b146be55962f239

Reply:
Ah, you're fixing it separately. 😃